webpack.config.js

```diff
@@ -0,0 +1,42 @@
+const path = require("path");
+const webpack = require("webpack");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+
+module.exports = function (env, options) {
+  const isProduction = options.mode === "production";
```

Comment:
I'd suggest using `env` here. See https://webpack.js.org/guides/environment-variables/ for details.